### PR TITLE
Add docs for budget and long-running run alerts

### PR DESCRIPTION
**Pull request by dpeng817**
As the title. Also documents the default policy functionality.

### CODE REVIEW

**Comment by dpeng817** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/22210?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#22210** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/22210?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#22206** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/22206?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @dpeng817 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: docs/content/dagster-plus/managing-deployments/alerts.mdx [hunk 1]
```diff
@@ -139,6 +144,23 @@ Alert policies are configured on a **per-deployment basis**. This means, for exa
...
     </tr>
+    <tr>
+      <td>
+        <strong>Credit budget alert (self-serve plans only)</strong>{" "}
+        <Experimental />
+      </td>
+      <td>
+        <strong>
+          This feature is currently only available for Dagster+ self-serve plans
+          (solo and starter).
```
Comment (erinkcochran87) [inline docs/content/dagster-plus/managing-deployments/alerts.mdx:155]:
```suggestion
          (Solo and Starter).
```

Plan names are a proper noun and should be capitalized

File: docs/content/dagster-plus/managing-deployments/alerts.mdx [hunk 2]
```diff
@@ -79,10 +79,15 @@ Alert policies are configured on a **per-deployment basis**. This means, for exa
...
-        will be eligible for that alert. Otherwise, only jobs that contain all
-        the tags for a given alert policy are eligible for that alert.
+        Triggers on job run success, failure, or time limit exceeded; may
+        optionally include a set of configured tags. If an alert policy has no
+        configured tags, all jobs will be eligible for that alert. Otherwise,
+        only jobs that contain all the tags for a given alert policy are
+        eligible for that alert.
+        <br />
+        Each newly created organization starts with a long-running run alert
+        policy, directed at the organization owner's email address, which will
```
Comment (erinkcochran87) [inline docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: > directed at the organization owner's email address, which will

Does this mean the user in the org who has the **Organization Admin** role?

**Review by erinkcochran87 — COMMENTED** [review]
One small nit and a ?, but looking good

File: docs/content/dagster-plus/managing-deployments/alerts.mdx (hunk 2 above)
Comment (dpeng817) [reply docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: There can be multiple organization admins, so by organization owner I hoped to disambiguate that from the person who initially created the organization (that's who is getting the alert).

Comment (erinkcochran87) [reply docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: Is there a way to identify this user in the UI? I quickly poked around a bit but didn't see anything obvious

Comment (dpeng817) [reply docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: There's no way to identify this user in the UI. It's just the person who goes through the onboarding flow initially.

Comment (erinkcochran87) [reply docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: Gotcha. I think we should consider adding a label or something for this in the UI. At a previous job, we had something similar for the app's billing contact and it lead to a ton of support questions and confusion.

Comment (dpeng817) [reply docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: That's a very good idea; I'll make a linear ticket for it (IMO we shouldn't block this PR on that though)

Comment (dpeng817) [reply docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: Gonna change it to just: the email address you originally created the organization with.

Comment (erinkcochran87) [reply docs/content/dagster-plus/managing-deployments/alerts.mdx:89]: Sounds good to me!

**Review by erinkcochran87 — APPROVED** [review]
Stamping this ahead of your last comment about the account's original email. Everything else looks good to me!

**Pull request merged**